client/components/blogging-prompt-card/prompts-navigation.jsx

```diff
@@ -158,11 +158,22 @@ const PromptsNavigation = ( { siteId, prompts, tracksPrefix, index } ) => {
 		return (
 			<div className="blogging-prompt__prompt-answers">
 				{ renderResponses() }
-				<Button href={ getNewPostLink() } onClick={ handleBloggingPromptClick }>
-					{ translate( 'Post Answer', {
-						comment:
-							'"Post" here is a verb meaning "to publish", as in "post an answer to this writing prompt"',
-					} ) }
+				<Button
+					href={ getNewPostLink() }
```

Comment:
In the case where the prompt is answered, should we link here to the editor to edit the post that is the answer rather than to ta new post?

Reply:
> should we link here to the editor to edit the post that is the answer
That could be good behavior if we know which post corresponds. Otherwise maybe just disabling the button is a good approach?

Reply:
yea we don't seem to have the post link handy, I'm not sure how we'd get it. I think it's better just to leave the link as it currently is.